rfc/18/README.md

```diff
@@ -39,7 +39,7 @@ nodes:
           data: [ <str> ]   # e.g. [ "/dev/loop0", "/dev/loop1", "/dev/loop2" ]
                             # Empty list means no IO service.
     m0_clients:
-        s3: <int>     # number of S3 servers to start
+        rgw: <int>     # number of rgw servers to start
```

Comment:
Refer to https://github.com/Seagate/cortx-`hare/blob/main/cfgen/examples/singlenode.yaml` correct format of "m0_clients:"

Reply:
Updated as per reference